### PR TITLE
Auth, Dashboard UI Changes

### PR DESCRIPTION
**Pull request by iamoperand**

### CODE REVIEW

**Comment by iamoperand** [comment]
## Authentication Screen
the remotants authentication

## Dashboard Screen
the remotants dashboard

## Error Notifications
the remotants error notifications

## Success Notifications
the remotants success notify

File: src/components/Login/Login.js [hunk 1]
```diff
@@ -3,32 +3,31 @@ import PropTypes from 'prop-types';
...
       if (!err) {
         const email = values.loginEmail;
         const password = values.loginPassword;
         try {
-          const { success } = await this.props.loginUser({ email, password });
-          if (success === true) {
+          const response = await this.props.loginUser({ email, password });
+          if (response.success === true) {
+            successNotify(response.message);
             this.props.history.push('/dashboard');
```
Comment (anshurathee) [inline src/components/Login/Login.js]: Use from route file.

File: src/components/Navbar/Navbar.js [hunk 2]
```diff
@@ -21,6 +21,15 @@ class NavBar extends React.Component {
             </NavLink>
           </Menu.Item>
         }
+        {
+          isAuthenticated &&
+          <Menu.Item key="dashboard">
+            <Icon type="dashboard" />
+            <NavLink style={{ display: 'inline' }} to="/dashboard">
```
Comment (anshurathee) [inline src/components/Navbar/Navbar.js]: Use from route file

File: src/config/routes.js [hunk 3]
```diff
@@ -4,4 +4,8 @@ const routes = {
   Dashboard: '/dashboard',
 };
 
+export const dashboardRoutes = {
```
Comment (anshurathee) [inline src/config/routes.js]: What is the need to export a blank object ?

File: src/components/App.js [hunk 4]
```diff
@@ -1,30 +1,42 @@
...
+import React from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import { Layout } from 'antd';
 
 import routes from '../config/routes';
 
 import PublicRoute from './PublicRoute';
 import PrivateRoute from './PrivateRoute';
-import Navbar from './Navbar';
+// import Navbar from './Navbar';
```
Comment (anshurathee) [inline src/components/App.js]: Why this line is commented ?

File: src/components/Dashboard/Dashboard.js [hunk 5]
```diff
@@ -1,14 +1,40 @@
...
+          </Breadcrumb>
+          <Content style={{
+            background: '#fff',
+            padding: 24,
+            margin: 0,
+            minHeight: 280,
+            }}
+          >
+            <Switch>
+              <PrivateRoute path="/dashboard/logout" component={Logout} />
```
Comment (anshurathee) [inline src/components/Dashboard/Dashboard.js]: Don't use embedded strings, use from route files

File: src/reducers/flashMessage.spec.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    expect(flashMessage(undefined, {})).toEqual({});
+  });
+  test('handle addFlashMessage with message provided', () => {
+    expect(flashMessage({}, {
+      type: flashMessageConstants.ADD_FLASH_MESSAGE,
+      message: {
+        type: 'success',
+        text: 'test',
+      },
+    })).toHaveProperty('id');
```
Comment (anshurathee) [inline src/reducers/flashMessage.spec.js]: Also check if the returned state has the same text which is sent.

**Pull request merged**